packages/asc-core/src/theme/default/typography.ts

```diff
@@ -26,7 +26,7 @@ const typography: Theme.TypographyInterface = {
     fontWeight: 700,
     letterSpacing: 'inherit',
     lineHeight: '28px',
-    marginBottom: '0',
+    marginBottom: '8px',
```

Comment:
have you checked the side effects of this change?

Reply:
doesn't affect anything in ASC, will add this to the changelist